playground/alex_bot.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+from random import choice
+from dataclasses import dataclass
+from typing import List, Tuple
+from board import Stone
+
+
+@dataclass
+class Window:
+    pattern: List[int]
+    position: List[Tuple[int, int]]
+
+
+class AlexBot:
+
+    def __init__(self, k: int = 6, m: int = 19, n: int = 19):
+        self.k = k
+        self.m = m
+        self.n = n
+
+    def put_stone(self, log: List[Stone]) -> Stone:
+        turn = "b" if (len(log) + 1) % 4 in [0, 1] else "w"
+        if len(log) == 0:
+            return Stone(str(10), str(10), turn)
+
+        if len(log) == 1:
+            next_x = int(log[0].x) + choice([-1, 1])
+            next_y = int(log[0].y) + choice([-1, 1])
```

Comment:
according to this code,
```python
next_x = int(log[0].x)
next_y = int(log[0].y) + choice([-1, 1])
```
or

```python
next_x = int(log[0].x) + choice([-1, 1])
next_y = int(log[0].y) 
```

won't be considered. Is this intended?

Reply:
Obviously, I missed it. Actually this is quite brief random logic, and I nearly spend time for here.